topics: ["/temperature", "/pressure"]
log_comment: "Sensor Data: "
log_path: "/home/alex/roslog/mylogs/"